driverdataaggregation/main.py

```diff
@@ -187,6 +167,13 @@ def aggregation_scaling(self,
 
             # stop database after this set of permutations have finished
             exp.stop(db)
+            #Added to clean up db folder bc of issue with exp.stop()
+            time.sleep(10)
```

Comment:
If you know what folder it is, you might be able to just repeatedly look for files with .rdb extension and delay until you don't find any

Reply:
sleep(10) shouldn't be needed

Reply:
yes so without the sleep, it loops through the if statement, gets rid of nothing, then by the time its done the db puts the file in the db folder, I put the sleep there bc it gives the db time and it did not work without it